servatrice/src/serversocketinterface.cpp

```diff
@@ -1905,11 +1902,11 @@ void WebsocketServerSocketInterface::initConnection(void *_socket)
     connect(socket, SIGNAL(disconnected()), this, SLOT(catchSocketDisconnected()));
 
     // Add this object to the server's list of connections before it can receive socket events.
-    // Otherwise, in case a of a socket error, it could be removed from the list before it is added.
+    // Otherwise, in case of a socket error, it could be removed from the list before it is added.
     server->addClient(this);
 
     logger->logMessage(
-        QString("Incoming websocket connection: %1 (%2)").arg(address.toString()).arg(socket->peerAddress().toString()),
+        QString("Incoming websocket connection: %1 (%2)").arg(address.toString(), socket->peerAddress().toString()),
```

Comment:
these are not the same
```suggestion
        QString("Incoming websocket connection: %1 (%2)").arg(address.toString()).arg(socket->peerAddress().toString()),
```